From: Johnnynator

errno on Musl and Glibc expand differently by one space
---
--- a/scripts/approvalTests.py	2019-12-28 21:24:04.000000000 +0100
+++ b/scripts/approvalTests.py	2019-12-28 21:24:04.000000000 +0100
@@ -44,7 +44,7 @@
 
 # errno macro expands into various names depending on platform, so we need to fix them up as well
 errnoParser = re.compile(r'''
-    \(\*__errno_location\ \(\)\)
+    \(\*__errno_location\s*\(\)\)
     |
     \(\*__error\(\)\)
     |
